### PR TITLE
First UI pull request

### PR DESCRIPTION
**Pull request by RaphaelAlvez**
Added Icons and Icons folder
Changed toolbar color to blue
Changed on hover of toolbar's buttons to blue
Changed font to Roboto

### CODE REVIEW

File: server/minify.mjs [hunk 1]
```diff
@@ -9,73 +10,73 @@ import uglifyES from '@node-minify/uglify-es';
...
+        }).then(function(min) {
+            roomHTML = min;
+            zlib.gzip(min, (err, gzipped) => {
+                resolve({ min, gzipped });
+            });
+        });
     });
-  });
-}
+}
```
Comment (UltimateGeek) [inline server/minify.mjs:82]: please restore the missing newline. And try to configure your editor to include newlines on the last line in text files.

File: server/minify.mjs [hunk 2]
```diff
@@ -9,73 +10,73 @@ import uglifyES from '@node-minify/uglify-es';
...
+                'client/css/widgets/button.css',
+                'client/css/widgets/card.css',
+                'client/css/widgets/classes.css',
+                'client/css/widgets/deck.css',
+                'client/css/widgets/holder.css',
+                'client/css/widgets/label.css',
+                'client/css/widgets/pile.css',
+                'client/css/widgets/spinner.css'
+            ],
+            output: os.tmpdir() + '/out.css'
```
Comment (UltimateGeek) [inline server/minify.mjs:36]: Do we really need a copy of the css & js written to files? This change of creating a tmp dir will now start creating new dirs that keep getting created and not cleaned up until the OS is rebooted.

File: client/css/layout.css [hunk 3]
```diff
@@ -12,11 +12,11 @@ body {
   padding: 0;
   margin: 0;
   overflow: hidden;
-  font-family: ubuntu, sans;
+  font-family: 'Roboto', sans-serif;
```
[the remark this answers is not in the document]
Comment (ArnoldSmith86) [reply client/css/layout.css:15]: OMG, `sans` was not correct and all you Windows people were seeing `serif` fonts?

File: client/css/layout.css [hunk 4]
```diff
@@ -70,11 +74,10 @@ body {
   padding: 2px;
   font-size: 14px;
   color: white;
-  background: black;
+  background: #1f5ca6;
```
[the remark this answers is not in the document]
Comment (ArnoldSmith86) [reply client/css/layout.css:77]: Also, if you intend to use the same color in multiple places, please use CSS variables (they are used for some colors already). That's not necessary if you only use it once though.

File: server/minify.mjs (hunk 2 above)
Comment (ArnoldSmith86) [reply server/minify.mjs:36]: The parameter isn't optional unfortunately. But os.tmpdir() shouldn't create a new directory. It just returns the default temp directory so it should be overwritten every time. There should probably be a better place for it though.

File: server/minify.mjs [hunk 5]
```diff
@@ -1,4 +1,5 @@
 import fs from 'fs';
```
Comment (ArnoldSmith86) [inline server/minify.mjs:1]: This whole file got pretty messed up by your editor. Like UltimateGeek said, please make sure to keep indentation at two spaces.

Also: the only change in this file is to replace /tmp by os.tmpdir. That fix is already commited to main. You should - I think - rebase this pull request to the latest revision of main.

**Review by ArnoldSmith86 — CHANGES_REQUESTED** [review]
Looks nice.

The way it is, you're committing the images as GPLv3. I don't know where they're coming from and I don't know if there license allows that.

**Comment by RaphaelAlvez** [comment]
> Looks nice.
> 
> The way it is, you're committing the images as GPLv3. I don't know where they're coming from and I don't know if there license allows that.

Yes, they have a license I just didn't know where to put their atribution.

**Comment by ArnoldSmith86** [comment]
You can play around with it here: http://212.47.248.129:7192/ukl9 (or any other room on that server)

**Pull request closed without merging**